plugins/scaffolder-actions/scaffolder-backend-module-aws/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@roadiehq/scaffolder-backend-module-aws",
+  "version": "0.1.0",
```

Comment:
Let's start with 1.0.0. It is better for users to know what is actually happening when we do modification and using semver properly is the way to do that.

Reply:
done